src/core_plugins/markdown_vis/public/markdown_vis_params.html

```diff
@@ -1,20 +1,43 @@
 <div class="kuiSideBarSection">
   <div class="markdown-vis-options form-group">
     <div>
-      <label for="markdownVisInput">Markdown</label>
-      <small class="pull-right"><a target="_window" href="https://help.github.com/articles/github-flavored-markdown/">Help <i aria-hidden="true" class="fa fa-link"></i></a></small>
+      <label
+        for="markdownVisInput"
+        i18n-id="markdownVis.public.markdownVisParams.markdownLabel"
+        i18n-default-message="Markdown"
+      >
+      </label>
+      <small class="pull-right">
+        <a target="_window" href="https://help.github.com/articles/github-flavored-markdown/">
+          <span
+            i18n-id="markdownVis.public.markdownVisParams.helpLinkLabel"
+            i18n-default-message="Help"
+          >
+          </span>
+          &nbsp;<i aria-hidden="true" class="fa fa-link"></i>
+        </a>
+      </small>
     </div>
     <div class="kuiSideBarFormRow">
       <label class="kuiSideBarFormRow__label" for="markdownVisFontSize">
-        Font Size (<span ng-bind="editorState.params.fontSize"></span>pt)
+        <span
+          i18n-id="markdownVis.public.markdownVisParams.fontSizeLabel"
+          i18n-default-message="Font Size"
+        >
+        </span>
+         (<span ng-bind="editorState.params.fontSize"></span>pt)
```

Comment:
Here I didn't translate `pt` because it`s a unit for measurement.
![image](https://user-images.githubusercontent.com/11524637/45553888-bdb3e780-b83d-11e8-8a16-93e362dd7ce7.png)

If there are any objections, let's discuss.